docs/basic-guide/cluster-configuration.mdx

```diff
@@ -150,11 +150,11 @@ Example
 
 ### Catalog service
 
-集群元数据相关配置
+Catalog service related configuration
 
 Keys:
 
-- **type** : 元数据存储引擎类型，支持 bytekv, fdb
+- **type** : Metadata storage engine type, support bytekv, fdb
```

Comment:
It's better to explain the meaning of fdb and bytekv. BTW I don't think bytekv is open-sourced.